pydu/request.py

```diff
@@ -137,3 +137,32 @@ def update_query_params(url, params):
     parts[4] = urlencode(query)
     new_url = urlparse.urlunparse(parts)
     return new_url
+
+
+class RequestException(Exception):
+    """Request Exception"""
+
+    pass
+
+
+def cookies_string_to_dict(cookies_string):
+    """
+    Transform cookies which is type of string  to the type of dict
+    """
+    if cookies_string is None or cookies_string == '':
+        raise RequestException("Invalidate param of cookies_string which is blank !")
+    if not isinstance(cookies_string, str):
+        raise RequestException("Invalidate param of cookies_string which is type of string")
```

Comment:
We should use `RequestException` because `cookies_string_to_dict` is only a convert func. 
* change previous `RequestException` to `ValueError`
* change this `RequestException` to `TypeError`